stack/stack_noaa.yaml

```diff
@@ -296,3 +296,28 @@ atlas:
 madis:
   build: YES
   version: 4.3
+
+esma_cmake:
+  build: YES
+  repo: GEOS-ESM
+  version: v3.4.3
+
+cmakemodules:
+  build: YES
+  repo: NOAA-EMC
+  version: develop
```

Comment:
we need to make a release, but until then, this is fine.
@kgerheiser Thoughts?

Reply:
A release of CMakeModules? Sure we can do that.

Reply:
@rmontuoro I made a release of CMakeModules could you update the version to `v1.2.0`?

Reply:
Done. See 4efc88048c529a90a54691bf0661140abbe75aa4.